src/Microsoft.Azure.WebJobs.Protocols/BlobParameterDescriptor.cs

```diff
@@ -5,19 +5,11 @@
 using Newtonsoft.Json;
 using Newtonsoft.Json.Converters;
 
-#if PUBLICPROTOCOL
```

Comment:
Can you describe the protocol changes here? I understand we're abandoning the attempt at having this project be a general protocol library for Dashboard publishing (since Dashboard is being obsoleted).